tests/utils/test_http_client.py

```diff
@@ -51,8 +51,8 @@ def test_reason(self):
 
         # use new session to disable retry
         cli = _HttpClient(session=Session())
-        # /error is an actual endpoint that exists on our service to test the behavior
-        res = cli.request("GET", "intake/error")
+        # /raise_error is an actual endpoint that exists on our service to test the behavior
+        res = cli.request("GET", "intake/raise_error")
```

Comment:
@yoshiori 

FYI: Updated this endpoint because you modified the endpoint if I understand correctly.

Reply:
I think this test is meaningless. 
Please remove this test case